clang/lib/Driver/Driver.cpp

```diff
@@ -5981,7 +5991,7 @@ class OffloadingActionBuilder final {
           SectionTriple += "-";
           SectionTriple += SyclTarget.BoundArch;
         }
-
+        SectionTriple = standardizedTriple(SectionTriple);
```

Comment:
If the bundler is doing the 'standardizing' to determine if the bundled sections are there - why do we need to do any standardizing on the caller side?

Reply:
Here, we are not calling the bundler. We seem to be reading the sections created by bundler directly.

Thanks

Reply:
Ah, I see - so here we are scanning all of the objects/archives and retrieving the section names.  These names could be names that are created with the updated standardized bundler behavior, requiring us to fix up any names for comparisons on the Driver end.